ci/docker_run.sh

```diff
@@ -12,16 +12,27 @@ if [ -z "$branch_specifier" ]; then
     IMAGE_NAME="$1"
 else
     # Jenkins
+    ci/docker_prune.sh
```

Comment:
there shouldn't be a need to call this explicitly. there is a seperate jenkins job that calls this.

Reply:
Will that run on all hosts though?

Reply:
it will eventually.  we only have 3 hosts today, and it will randomly run on 1 of the 3 every hour iirc. "eventually clean" , like me in my youth.